.babelrc

```diff
@@ -12,7 +12,6 @@
     }]
   ],
   "plugins": [
-    "transform-object-rest-spread",
```

Comment:
@babel/proposal-object-rest-spread is included in @babel/preset-env, so I quit specifying it explicitly